runtime-modules/content/src/lib.rs

```diff
@@ -765,15 +779,12 @@ decl_module! {
             Self::ensure_channel_bag_can_be_dropped(channel_id, num_objects_to_delete)?;
 
             // try to remove the channel
-            Self::try_to_perform_channel_deletion(sender.clone(), channel_id)?;
+            Self::try_to_perform_channel_deletion(&sender, channel_id, channel)?;
```

Comment:
Currently it's still returned, I planned to fix this as part of https://github.com/Joystream/joystream/issues/4208, but I can do it within the scope of this PR, since it's a small change

Reply:
Done in https://github.com/Joystream/joystream/pull/4164/commits/deaf7f35d2f97f1068c51df5364962d755b043b2